server/http_handler.go

```diff
@@ -1010,14 +1020,22 @@ func (h tableHandler) getRegionsByID(tbl table.Table, id int64, name string) (*T
 		indices[i].Name = index.Meta().Name.String()
 		indices[i].ID = indexID
 		startKey, endKey := tablecodec.GetTableIndexKeyRange(id, indexID)
-		rIDs, err := h.RegionCache.ListRegionIDsInKeyRange(tikv.NewBackoffer(context.Background(), 500), startKey, endKey)
+		regions, peers, err := pdCli.ScanRegions(ctx, startKey, endKey, 0)
```

Comment:
Could you unify the limit to 1 or 0?

Reply:
-1 ?

Reply:
Yes. Either -1 or 0 is ok, but not use them both.

Reply:
Now I change them to -1